zsh-syntax-highlighting.zsh

```diff
@@ -28,13 +29,17 @@
 # -------------------------------------------------------------------------------------------------
 
 # First of all, ensure predictable parsing.
-zsh_highlight__aliases=`builtin alias -Lm '[^+]*'`
+local zsh_highlight__aliases="`builtin alias -Lm '[^+]*'`"
 # In zsh <= 5.2, `alias -L` emits aliases that begin with a plus sign ('alias -- +foo=42')
 # them without a '--' guard, so they don't round trip.
 #
 # Hence, we exclude them from unaliasing:
 builtin unalias -m '[^+]*'
 
+# Set the 5.4-to-be WARN_NESTED_VAR option if it's available.
+emulate -L zsh
+setopt localoptions warncreateglobal ${(k)options[(I)warnnestedvar]}
```

Comment:
`$options` requires zsh/parameter. Perhaps you could use `zsyh_user_options`?

Reply:
Touché.